packages/frontend/src/models/Transaction.ts

```diff
@@ -133,6 +118,53 @@ class Transaction extends EventEmitter {
     }
   }
 
+  getRpcUrl(networkName): string {
+    let rpcUrl = ''
+    if (networkName.startsWith('arbitrum')) {
+      rpcUrl = getRpcUrl('arbitrum')
+    } else if (networkName.startsWith('optimism')) {
+      rpcUrl = getRpcUrl('optimism')
+    } else if (networkName.startsWith('xdai')) {
+      rpcUrl = getRpcUrl('xdai')
+    } else if (networkName.startsWith('matic') || networkName.startsWith('polygon')) {
```

Comment:
I think it's safe to remove `networkName.startsWith('matic')` and just have `networkName.startsWith('polygon')`